### DISCUSSION TITLE
UNderstanding background in confusion matrix

### DISCUSSION THREAD

**Discussion by unrue** [ultralytics/yolov5#12724] [General]
I'm training yolo on a dataset having 30.000 and 75 classes, v5x model.

The following is my actual confusion matrix. I note a huge number of blue box in background class. It means I have a lot of false positive? How can I reduce that? Any suggestions? Thanks.

![confusion_matrix](https://github.com/ultralytics/yolov5/assets/51996285/b4613abe-9502-46a5-9533-d18b3369c3db)

**Comment by jmarintur** [comment]
Hi @unrue, the confusion matrix is telling you that your model is significantly failing at detecting bananas, fish, hands or oranges. For these classes, you have a lot of false negatives, since the detector is failing at detecting true positive cases.

Overall, having a lot of cases that fall into the background class is just telling you that your detector was not able to detect them. As you see, this doesn't happen for all the classes, you detect zebras or pegasus pretty well for instance. 

About improving your model, I'd suggest first to check if the model performance already saturated on your validation dataset, if not, keep training it until it does.
